app/src/main/java/com/github/libretube/ui/sheets/PlaylistOptionsBottomSheet.kt

```diff
@@ -105,8 +114,20 @@ class PlaylistOptionsBottomSheet(
                 }
 
                 getString(R.string.change_playlist_description) -> {
-                    PlaylistDescriptionDialog(playlistId, "", onChangeDescription)
-                        .show(parentFragmentManager, null)
+                    val bundle = bundleOf(
+                        IntentData.playlistId to playlistId,
+                        IntentData.currentPlaylistDescription to ""
+                    )
+                    val newShareDialog = PlaylistDescriptionDialog()
+                    newShareDialog.arguments = bundle
+                    newShareDialog.show(parentFragmentManager, null)
+                    parentFragmentManager.setFragmentResultListener(
+                        "requestKey",
```

Comment:
Please save that to a constant somewhere (it'd be okay for me to add it to `IntentData` too.

Reply:
The `requestkey` ?

Reply:
Exactly